apps/web/src/server/api/routers/organization.ts

```diff
@@ -29,14 +30,19 @@ export const organizationRouter = createTRPCRouter({
   }),
 
   createOrganization: protectedProcedure
-    .input(
-      z.object({
-        name: z.string().max(128),
-      }),
-    )
+    .input(insertOrganizationSchema.omit({ owner: true }))
     .mutation(async ({ ctx, input }) => {
       await ctx.db
         .insert(organizations)
         .values({ name: input.name, owner: ctx.session.user.id });
+
+      const newOrganization = (
+        await ctx.db
+          .select()
+          .from(organizations)
+          .where(eq(organizations.name, input.name))
+      ).at(0);
+
+      return newOrganization;
     }),
```

Comment:
Consider changing this to:

```suggestion
    .input(insertOrganizationSchema.omit({ owner: true }))
    .mutation(async ({ ctx, input }) => {
      const newOrganization = await ctx.db
        .insert(organizations)
        .values({ name: input.name, owner: ctx.session.user.id })
        .returning();

      return newOrganization;
    }),
```
MySQL don't have this functionality since we had to make additional query to retrieve newly added reosource.